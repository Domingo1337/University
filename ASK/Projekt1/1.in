add 	$zero,$s2,$t3 	#nazwy rejestrów
addi 	$1, $2, -17	#stałe
xori 	$1, $2, 0xF0D1
