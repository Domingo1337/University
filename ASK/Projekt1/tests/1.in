add	$zero,$s2,$t3 	#nazwy rejestrow
addi 	$s1, $sp, -17	#stale
xori 	$1 $2,0xF0D1	#rozne sposoby podawania argumentow
