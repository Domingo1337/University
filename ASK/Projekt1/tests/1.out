.text
00000000	024b0020	add	$0,$18,$11
00000004	23b1ffef	addi	$17,$29,-17
00000008	3841f0d1	xori	$1,$2,0xF0D1
